### DISCUSSION TITLE
Anisotropic filtering blurs things and makes fishing spots hard to see?

### DISCUSSION THREAD

**Discussion by Espionage724** [runelite/runelite#16450] [Q&A]
Is Anisotropic filtering/AF supposed to bring a benefit? I've noticed it blurs textures like on wooden doors, trees and if I recall right worn Void or Graceful armor, and makes the white specks on fishing spots in water nearly-invisible. This happens at any level above 0.

**Comment by ghost** [comment]
When you got anisotropic filter off you will notice sharp edges on everything in the world.
Increasing it will decrease the edges, smoothing them out.

For example trees, whenever you move and you got this on you no longer see the sharp outlines on them.

Watch out you don't have options on your screen which work against each other...
Sharpening filter, to say one of them

**Reply by Espionage724** [reply]
You may be confusing that with AA. That works fine for smoothing edges. AF is what I'm talking about it and it seems to offer no benefit to even be an option.
